src/pages/login/login.html

```diff
@@ -50,7 +50,7 @@
     </div>
 
   </form>
-  <button ion-button (click)="goForgotPassword()" clear>Mot de passe oublié</button>
+  <button ion-button (click)="ForgotPassword()" clear>Mot de passe oublié</button>
```

Comment:
caution, you have a function that starts with an Uppercase. This is reserved to the classes